components/econet/climate/__init__.py

```diff
@@ -21,6 +21,8 @@
 CONF_MODE_DATAPOINT = "mode_datapoint"
 CONF_CUSTOM_PRESET_DATAPOINT = "custom_preset_datapoint"
 CONF_CUSTOM_FAN_MODE_DATAPOINT = "custom_fan_mode_datapoint"
+CONF_CUSTOM_FAN_MODE_NO_SCHEDULE_DATAPOINT = "custom_fan_mode_no_schedule_datapoint"
+CONF_CUSTOM_FOLLOW_SCHEDULE_DATAPOINT = "custom_follow_schedule_datapoint"
```

Comment:
Should this be `custom_fan` or `custom_fan_mode` instead of `custom`? Or just drop the `custom` and make it `follow_schedule_datapoint` since the object is SCHEDULE? It would also match the `follow_schedule` variable you have in the C++ code.

Reply:
Updated to follow_schedule_datapoint in latest commit.